build-win32.txt

```diff
@@ -3,7 +3,6 @@ c = 'i686-w64-mingw32-gcc'
 cpp = 'i686-w64-mingw32-g++'
 ar = 'i686-w64-mingw32-ar'
 strip = 'i686-w64-mingw32-strip'
-cmake = 'i686-w64-mingw32-cmake'
```

Comment:
Tried rebasing? This commit should already be on master.

Reply:
IDK how this got caught up in this series...

Will do

Reply:
Resolved.